Test/XLTestNGlyco.cs

```diff
@@ -75,7 +75,7 @@ public static void GlyTest_GlyGetTheoreticalFragments()
             //    }
             //}
 
-            CommonParameters commonParameters = new CommonParameters(deconvolutionMassTolerance: new PpmTolerance(20));
+            CommonParameters commonParameters = new CommonParameters(deconvolutionMassTolerance: new PpmTolerance(20), trimMsMsPeaks: false);
```

Comment:
I added peak filtering for .mgf files to the new mzLib version, which was missing. I edited this unit test to not filter MS2 peaks to keep the results of the test the same as the old version.